enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotQuestion.tsx

```diff
@@ -37,7 +37,6 @@ const MetabotQuestionInner = () => {
         <InteractiveAdHocQuestion
           questionPath={redirectUrl}
           title={false}
-          onNavigateBack={() => {}}
```

Comment:
No longer needed as we hide the button, instead of rendering a button that does nothing.